package.json

```diff
@@ -40,7 +40,7 @@
     "lodash": "^3.10.0",
     "method-override": "^2.3.3",
     "mocha": "~1.20.0",
-    "mongoose": "^4.0.6",
+    "mongoose": "^4.1.11",
```

Comment:
!fix  You need to be careful about pushing up changes to package.json. I know this is to fix the whole mongodb auth issue but make sure you let everyone know that you are going to do this because if you pushed up a different change to a working package it could break more things than it fixes!